storage/spanstore/memory/memory.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I needed the memory store to be available for the standalone main function. I agree that this diff is accomplishing multiple tasks, I'd like to get away with it this time (if you don't mind) instead of backtracking to break up the diff.